9.2/paper.tex

```diff
@@ -294,8 +294,8 @@ \subsection{A new fully distributed triangulation class}
 
 Previously, all triangulation classes of \texttt{deal.II} had in common that the coarse grid is replicated by
 all processes in a parallel environment, and the actual mesh used for computations is constructed by repeated
-refinement. However, this has its limitations in industrial
```

Comment:
I changed since, for instance, I have this kind of configuration and it is an academic problem.